backend/company-agent/src/main/java/org/hyperledger/oa/api/aries/SchemaAPI.java

```diff
@@ -46,4 +49,8 @@ public static SchemaAPI from(BPASchema s) {
                 .schemaId(s.getSchemaId())
                 .build();
     }
+
+    public void addSchemaAttributes(Set<String> schemaAttributeNames) {
```

Comment:
This is a a setter. Add implies either adding a single element to the set or merging the sets.